strongbox-cron/strongbox-cron-api/src/main/resources/etc/conf/strongbox-cron-tasks.yaml

```diff
@@ -0,0 +1,37 @@
+cronTasksConfiguration:
+  - uuid: 463ff872-b7b2-4cb9-aff9-37b3baed4f14
+    name: Clear Each Repository's Trash Cron Job
+    oneTimeExecution: false
+    immediateExecution: false
+    properties:
+      cronExpression: 0 0 1 * * ?
+      jobClass: org.carlspring.strongbox.cron.jobs.ClearRepositoryTrashCronJob
+  - uuid: 2222a559-6eac-4cda-81d2-83dbfcf1b1e0
+    name: Cleanup Expired Artifacts From Proxy Repositories Cron Job
+    oneTimeExecution: false
+    immediateExecution: false
+    properties:
+      cronExpression: 0 0 1 * * ?
+      jobClass: org.carlspring.strongbox.cron.jobs.CleanupExpiredArtifactsFromProxyRepositoriesCronJob
+      lastAccessedTimeInDays: 60
+  - uuid: 9dd9a6ef-d953-4ce0-be3d-3031fcfd993b
+    name: Re-generate Artifacts Checksums Cron Job
+    oneTimeExecution: false
+    immediateExecution: false
+    properties:
+      cronExpression: 0 0 1 * * ?
+      jobClass: org.carlspring.strongbox.cron.jobs.RegenerateChecksumCronJob
+  - uuid: b44164a5-c900-43ff-8d13-7a0d0e509d1d
+    name: Re-build All Maven Repositories Metadata Files Cron Job
+    oneTimeExecution: false
+    immediateExecution: false
+    properties:
+      cronExpression: 0 0 1 * * ?
+      jobClass: org.carlspring.strongbox.cron.jobs.RebuildMavenMetadataCronJob
+  - uuid: c9e7eeef-4e97-4500-aa12-e714a246c2dc
+    name: Remove Timestamped Maven Snapshots Cron Job
+    oneTimeExecution: false
+    immediateExecution: false
+    properties:
+      cronExpression: 0 0 1 * * ?
+      jobClass: org.carlspring.strongbox.cron.jobs.RemoveTimestampedMavenSnapshotCronJob
```

Comment:
Cosmetic: Could you please add a new line at the end of such configuration resources? Thanks!